src/main/bash/commands/kobman-install.sh

```diff
@@ -21,11 +21,10 @@ function __kobman_validate_environment
 }
 
 function __kobman_validate_version_format
```

Comment:
@asa1997 Can this function be moved to environment helper?

Reply:
It has been moved to env helper along with other validation functions